src/System.Private.Xml/tests/XmlSerializer/XmlSerializerTests.cs

```diff
@@ -4395,6 +4395,149 @@ public static void DerivedTypeWithDifferentOverrides()
     }
 #endif
 
+    [Fact]
+    public static void Xml_DefaultNamespaceChange_SimpleArray_ObjectAsRoot()
+    {
+        SimpleType[] x = new SimpleType[] { new SimpleType { P1 = "abc", P2 = 11 }, new SimpleType { P1 = "def", P2 = 12 } };
+#if uap
```

Comment:
After the change, can you please try running the test targeting `uapaot`?  I suspected the line below would fail the ReflectionOnly test project.

If it fails, we probably can change the above `#if` to `#if uapaot && !ReflectionOnly`.